docs/release_notes.md

```diff
@@ -1,5 +1,30 @@
 # Release Notes
 
+## Version 1.3.0
+
+### Core
+
+- Implemented a new solver for fat simulation.
+- Implemented a new constraint to preserve the internal shape of volumetric geometries.
+- Implemented a new constraint to preserve the volume in tetrahedralized geometries.
+- Extended all existing solvers to support stiffness overrides per constraint type.
+
+### Maya
```

Comment:
Should we add a note about the AdonisFX shelf loading issue?

Reply:
Also the damp velocities fix?

Reply:
Thanks for the suggestions. Both are applied!